dlhub_sdk/client.py

```diff
@@ -16,19 +13,50 @@
 import os
 
 
-class DLHubClient:
+# TODO: Make this inherit from BaseClient in globus_sdk
+class DLHubClient(BaseClient):
     """Main class for interacting with the DLHub service
 
     Holds helper operations for performing common tasks with the DLHub service. For example,
     `get_servables` produces a list of all servables registered with DLHub."""
 
-    def __init__(self, timeout=None):
+    def __init__(self, authorizer, http_timeout=None, **kwargs):
         """Initialize the client
 
         Args:
-            timeout (int): Timeout for any call to service. (default is no timeout)
-            """
-        self.timeout = timeout
+            authorizer (:class:`GlobusAuthorizer <globus_sdk.authorizers.base.GlobusAuthorizer>`):
+                An authorizer instance used to communicate with DLHub
+            http_timeout (int): Timeout for any call to service in seconds. (default is no timeout)
+        Keyword arguments are the same as for BaseClient
+        """
+        super(DLHubClient, self).__init__("DLHub", environment='dlhub', authorizer=authorizer,
+                                          http_timeout=http_timeout, base_url=DLHUB_SERVICE_ADDRESS,
+                                          **kwargs)
+
+    @classmethod
+    def login(cls, force=True, **kwargs):
```

Comment:
Maybe force should be `false` by default?

Reply:
Yea, it totally should. I'll fix that